### PR TITLE
fix: Remove https from domain and default delegationNetwork to mainnet

### PR DESCRIPTION
**Pull request by ChaituVR**
Fixes https://github.com/snapshot-labs/workflow/issues/253

## How to test

- In `src/helpers/poke.ts` comment the code that checks validation: (until we merge https://github.com/snapshot-labs/snapshot.js/pull/1074)
```ts
// if (snapshot.utils.validateSchema(snapshot.schemas.space, space) !== true) {
    //   return Promise.reject('invalid space format');
    // }
```
- Open http://localhost:3001/spaces/cow.eth/poke
- Should save space settings without any error (domain should be saved as `vote.cow.fi`)
- In `src/helpers/actions.ts` add `delete settings.domain` on line `10`
- Open http://localhost:3001/spaces/cowtesting.eth/poke 
- Should save space settings without any error

### CODE REVIEW

File: src/helpers/actions.ts [hunk 1]
```diff
@@ -4,6 +4,15 @@ import { DEFAULT_NETWORK_ID, jsonParse, NETWORK_ID_WHITELIST } from './utils';
 
 export async function addOrUpdateSpace(space: string, settings: any) {
   if (!settings?.name) return false;
+  if (settings.domain) {
+    settings.domain = settings.domain?.replace(/(^\w+:|^)\/\//, '').replace(/\/$/, '');
+  }
+  if (settings.delegationPortal) {
+    settings.delegationPortal = {
+      ...settings.delegationPortal,
+      delegationNetwork: settings.delegationPortal?.delegationNetwork ?? '1'
```
Comment (ChaituVR) [inline src/helpers/actions.ts:13]:
```suggestion
      delegationNetwork: settings.delegationPortal.delegationNetwork ?? '1'
```

**Review by bonustrack — APPROVED** [review]

File: src/helpers/actions.ts [hunk 2]
```diff
@@ -4,6 +4,15 @@ import { DEFAULT_NETWORK_ID, jsonParse, NETWORK_ID_WHITELIST } from './utils';
 
 export async function addOrUpdateSpace(space: string, settings: any) {
   if (!settings?.name) return false;
+  if (settings.domain) {
+    settings.domain = settings.domain?.replace(/(^\w+:|^)\/\//, '').replace(/\/$/, '');
+  }
+  if (settings.delegationPortal) {
+    settings.delegationPortal = {
+      ...settings.delegationPortal,
+      delegationNetwork: settings.delegationPortal.delegationNetwork ?? '1'
```
Comment (wa0x6e) [inline src/helpers/actions.ts:13]: Swapping these 2 lines would have allowed using just `delegationNetwork: '1'`

Comment (ChaituVR) [reply src/helpers/actions.ts:13]: you mean like this? 
```suggestion
      delegationNetwork: '1',
      ...settings.delegationPortal      
```

Comment (ChaituVR) [reply src/helpers/actions.ts:13]: https://github.com/snapshot-labs/snapshot-sequencer/pull/462

**Pull request merged**